Xamarin.PropertyEditing.Windows/PropertyButton.cs

```diff
@@ -145,10 +127,14 @@ private void OnValueSourceChanged (ValueSource source)
 					break;
 				case ValueSource.DefaultStyle:
 				case ValueSource.Style:
-				case ValueSource.Resource:
 					ToolTip = null;
 					brush = "PropertyResourceBrush";
 					break;
+				case ValueSource.Resource:
+					// VS actually says "static" for system resources, but I think that's worth breaking with
+					ToolTip = (this.vm != null) ? String.Format (Properties.Resources.ResourceWithName, this.vm.Resource?.Name) : Properties.Resources.Resource;
```

Comment:
Might be a little better to use the Resource string also if `vm.Resource?.Name` is null or empty.